NumberPlayScanner/src/Main.java

```diff
@@ -0,0 +1,15 @@
+import org.opencv.core.Core;
+import org.opencv.imgcodecs.*;
```

Comment:
I'd prefer separate imports (like `import org.opencv.imgcodecs.Imgcodecs;`) to a wildcard import.